delivery_recipe/technical-delivery/code-quality/README.md

```diff
@@ -0,0 +1,85 @@
+# Code Quality
+
+Code quality is a way of referring to the quality of the product being built and the way that is being built – the product, the architecture, the design, the codebase, the technical team. It is a larger idea that covers many aspects of best practice when programming, when working in a technical team to develop products.
+    
+In that context, there's many best practices, methodologies, conventions and rules of thumb that we subscribe to here at Convivio.
+ 
+## Don't Write Code (write new code only when everything else fails)
+
+> This is the single most important lesson every developer needs to learn. The amount of duplicate, crappy code (across projects) that exists today is overwhelming. In a lot of cases developers don’t even bother to look around. They just want to write code.
+
+(**Source:** [Naresh Jain, 'Biggest Stinkers', *Managed Chaos blog*](http://blogs.agilefaqs.com/2009/10/19/biggest-stinkers/))
+
+### Open source it
+
+The manifestation of the 'Don't Write Code' mantra is that _we believe in [open source software](https://opensource.com/resources/what-open-source) (OSS)_. By using OSS we are able to benefit from the uncountable hours of design and programming already invested by developers all over the world, and the only new code written is the necessary code. 
+
+We use OSS wherever possible, especially in product development, and consequently we participate in the open source community. Consequently and intentionally, we give back to the OSS community – sharing work, offering and reviewing patches for features or bugs, sharing knowledge, and so on. 
+
+We use [Drupal](https://www.drupal.org/) for many projects so our largest interaction is with the Drupal community, but we also participate in the wider PHP community, as well as the frontend Javascript technology communities and others.
+
+## Quality matters
+
+> When I hear "JUST BANG OUT CODE THAT WORKS" I think of all the apps I don't use anymore because they gradually lost the ability to iterate.
+
+(**Source:** [Avdi Grimm](https://twitter.com/#!/avdi/status/180747721852985344))
+
+**working code > not-working code**
+ 
+_But_, it is better to work within the coding conventions of the technology you are using as this will tend towards high quality code. Coding conventions are there for good reasons:
+
+* they make code run well;
+* they make code readable when other developers come to use it;
+* conventions are often hard-won through brutal experience of writing efficient, effective portable code.
+
+For example, when working on Drupal projects we subscribe to the community-agreed [Drupal coding standards](https://www.drupal.org/docs/develop/standards).
+
+Code changes should always be reviewed. This includes:
+
+* peer review, through, for example, pull requests;
+* [testing](/delivery_recipe/technical-delivery/testing/README.md), both:
+  * automated testing, and,
+  * user testing.
```

Comment:
Do we have a place for design/UX review? I think that's slightly different to user testing